src/components/Search/SearchRouter/SearchRouterList.tsx

```diff
@@ -148,6 +148,8 @@ function SearchRouterList(
             {
                 excludeLogins: CONST.EXPENSIFY_EMAILS,
                 includeSelfDM: true,
+                showChatPreviewLine: true,
+                shouldBoldTitleByDefault: false,
```

Comment:
This fixes: https://github.com/Expensify/App/issues/53289

Reply:
This is the PR that originally introduced `shouldBoldTitleByDefault`: https://github.com/Expensify/App/pull/48575/files

I understand that we used to pass `shouldBoldTitleByDefault: false` before the refactor https://github.com/Expensify/App/pull/52568